src/lilURL.php

```diff
@@ -61,13 +69,46 @@ public function handleRedirect($id)
         if ($id != '' && $id != basename($_SERVER['PHP_SELF']) && $id != '?login') {
             $location = $this->getURL($id);
             if ($location != false) {
+                $this->trackHit($id);
```

Comment:
Shucks. The library should update their documentation to reflect that. :P Also, re our out of band conversation: 1) lowering the timeout would be good, but it isn't configurable and 2) it doesn't look like google doesn't allow you to customize a request date so the cron idea won't work ideally.

If this DOES turn into a real issue, we could always implement our own basic analytics tracking (simple mysql table with a row for each request).